src/data/navigation/sections/checkout-starter-kit.js

```diff
@@ -1,57 +1,61 @@
 module.exports = [
-    {
-        title: "Checkout starter kit",
-        header: true,
+  {
+    title: "Checkout starter kit",
+    header: true,
+    path: "/starter-kit/checkout/index.md",
+    pages: [
+      {
+        title: "Introduction",
         path: "/starter-kit/checkout/index.md",
-        pages: [
```

Comment:
Why are we removing the pages section?

Reply:
No, is not removed, just modified, check the diff:
![image](https://github.com/user-attachments/assets/5463cfe7-6f2a-4815-9150-8d560dd3c90d)